src/components/ReactP5WrapperWithSketch.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { createRef, useEffect, useRef } from "react";
+import React, { useEffect, useRef } from "react";
 
```

Comment:
We don't use "createRef", so it can be gone, right?

Reply:
It's gone 😉!